Psychtoolbox/PsychFiles/PsychHomeDir.m

```diff
@@ -101,7 +93,11 @@
 if exist('subDir', 'var')
     % Yes. Usercode wants path to subdirectory inside config dir. Assemble
     % path name:
-    ThePath = [ThePath subDir];
+	if ThePath(end) ~= filesep
+		ThePath = [ThePath subDir];
```

Comment:
i'm a bit tired, but this looks backwards? then and else branch switched?

Reply:
Indeed I made a quick commit to fix this in 759fb1d